graph-builder/src/config/options.rs

```diff
@@ -75,6 +75,10 @@ pub struct DockerRegistryOptions {
     /// Metadata key where to record the manifest-reference
     #[structopt(long = "upstream.registry.manifestref_key")]
     pub manifestref_key: Option<String>,
+
+    /// Concurrency for graph fetching
+    #[structopt(long = "upstream.registry.fetch_concurrency")]
```

Comment:
Lets update a configmap in `dist/openshift/cincinnati.yaml` with the default value so that it would not be hidden in the source code